OWNERS

```diff
@@ -1,3 +1,5 @@
+component: "Monitoring"
```

Comment:
What does this mean? This is not part of the official [OWNERS file spec](https://github.com/kubernetes/community/blob/master/contributors/guide/owners.md) and it is very informative.

Reply:
nvm, I need to read emails first :D
/lgtm